apps/prairielearn/src/admin_queries/courses_with_chunks.ts

```diff
@@ -1,4 +1,9 @@
-import { runLegacySqlAdminQuery } from './util.js';
+import { type AdministratorQuerySpecs, runLegacySqlAdminQuery } from './lib/util.js';
+
+export const specs: AdministratorQuerySpecs = {
+  description: 'Finds all courses that have chunks generated.',
+  params: [],
```

Comment:
This was copied straight from the JSON, which had an explicit empty list instead of no params argument at all. In practice, this means that this query is not run automatically on load, since that logic checks for nullish, not empty. I don't see a reason for keeping the empty, but I don't want to make assumptions here.

https://github.com/PrairieLearn/PrairieLearn/blob/3eb7435c50dc29a60331f5530cfc5426472b08d5/apps/prairielearn/src/pages/administratorQuery/administratorQuery.ts#L31-L35

Reply:
Jumping in since Peter tagged me for a look: I don't really care either way, but honestly we could probably just remove this query entirely. It was only useful when we were rolling out chunks for the first time; we'd never have to use this now.

If you want to land a PR removing this first, we can avoid needing to make a decision here 🙂